src/baselaplace/predicting.jl

```diff
@@ -1,3 +1,27 @@
+using Distributions
```

Comment:
This is the place where you currently import the entire namespace, so let's be more specific here to avoid the warning :)